chrome_manifest_v3/model.js

```
@@ -1,784 +1,764 @@
-    }
+  if (this.pos !== 'at') {
+    this.data.push({ property: 'cards' }); // Seed array for new card
+  }
 };
 
 Gmail2Trello.Model.prototype.Uploader.prototype = {
-    attachments: "attachments",
+  attachments: 'attachments',
+
+  exclude: function (list, exclude) {
+    let list_new = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'list_new' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)